src/discussions/comments/comment/Comment.jsx

```diff
@@ -0,0 +1,40 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import CommentIcons from '../comment-icons/CommentIcons';
+
+function Comment({ comment }) {
+  return (
+    <div className="discussion-comment d-flex flex-column">
+      <div className="header d-flex flex-row">
+        <div className="d-flex flex-column flex-fill">
+          <div className="title h4">
+            { comment.title }
+          </div>
+          <div className="status small">
```

Comment:
Good to go as is, but I feel we're lacking an even smaller font size for a few things (CC @xitij2000).

Reply:
Noted, we can suggest a new size at the paragon level.